roles/pulp_common/tasks/repos.yml

```diff
@@ -126,14 +126,23 @@
     - ansible_facts.distribution == "CentOS"
     - ansible_facts.distribution_major_version|int >= 8
 
-- name: "Add {{ __pulp_pkg_repo_name }} RPM repository"
-  yum_repository:
-    name: "{{ __pulp_pkg_repo_name }}"
-    description: "{{ __pulp_pkg_repo_name }}"
-    baseurl: "{{ pulp_pkg_repo }}"
-    gpgcheck: "{{ pulp_pkg_repo_gpgcheck }}"
-    gpgkey: "{{ __pulp_pkg_repo_gpgkey }}"
-    enabled: true
+- name: "Configure {{ __pulp_pkg_repo_name }} RPM repository"
+  block:
+    - name: "Create module_hotfixes config file"
+      copy:
+        dest: /etc/yum/pulpcore.conf
+        content: |
+          module_hotfixes=1
```

Comment:
Please add a comment as to why we need `module_hotfixes=1`, and why the `copy` / `include` workaround is needed.

Put it above the copy task's `name`.

Reply:
done